### PR TITLE
OpenBLAS-2.20 and UWP support

### PR DESCRIPTION
**Pull request by neilsh-msft**
Update to support OpenBLAS 0.2.20 and x64-UWP build support.

https://github.com/Microsoft/vcpkg/issues/1606

### CODE REVIEW

File: ports/openblas/portfile.cmake [hunk 1]
```diff
@@ -35,20 +34,63 @@ vcpkg_apply_patches(
...
+set(ENV{PATH} "$ENV{PATH};${PERL_EXE_PATH};${SED_EXE_PATH}")
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    OPTIONS -DTARGET=NEHALEM -DBUILD_WITHOUT_LAPACK=ON
-    # PREFER_NINJA # Disable this option if project cannot be built with Ninja
-    # OPTIONS -DUSE_THIS_IN_ALL_BUILDS=1 -DUSE_THIS_TOO=2
-    # OPTIONS_RELEASE -DOPTIMIZE=1
-    # OPTIONS_DEBUG -DDEBUGGABLE=1
+message(STATUS "woo: ${TARGET_TRIPLET} ${VCPKG_CMAKE_SYSTEM_NAME}")
```
Comment (KindDragon) [inline ports/openblas/portfile.cmake]: woo?

Comment (neilsh-msft) [inline ports/openblas/portfile.cmake]: Development debug output, removed.

File: ports/openblas/portfile.cmake [hunk 2]
```diff
@@ -65,7 +105,9 @@ string(REPLACE "#include \"common.h\"" "#include \"openblas_common.h\"" CBLAS_H
 file(WRITE ${CURRENT_PACKAGES_DIR}/include/cblas.h "${CBLAS_H}")
 
 # openblas is BSD
-file(COPY ${CURRENT_BUILDTREES_DIR}/src/OpenBLAS-0.2.19/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/openblas)
+file(COPY ${CURRENT_BUILDTREES_DIR}/src/OpenBLAS-0.2.20/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/openblas)
```
Comment (KindDragon) [inline ports/openblas/portfile.cmake]: I think maybe better use SOURCE_PATH here?

**Comment by neilsh-msft** [comment]
Updated to use SOURCE_PATH

**Comment by alexkaratarakis** [comment]
Thanks! 

Note that the build fails if there are spaces in the path (not a regression though, it didn't work before either).
I added it to the list of #807.

**Pull request merged**